web/packages/teleport/src/Roles/RoleEditor/StandardEditor/Resources.tsx

```diff
@@ -537,7 +540,14 @@ export function DatabaseAccessSection({
         value={value.roles}
         onChange={roles => onChange?.({ ...value, roles })}
         rule={precomputed(validation.fields.roles)}
-        mb={0}
+      />
+      <LabelsInput
+        legend="Database Service Labels"
+        tooltipContent="The database service labels have no influence on which databases' access is controlled by this role. Instead, they control which database services are discoverable while enrolling a new database."
```

Comment:
> Instead, they control which database services are discoverable while enrolling a new database.

tbh, i didn't understand what this means 😢, can we simplify it? The ai doc says:

<img width="651" alt="image" src="https://github.com/user-attachments/assets/84431f79-a5fa-428e-9a39-212277a1c812" />

Reply:
@kimlisa Take a look at [this discussion](https://gravitational.slack.com/archives/C0DF0TPMY/p1737117092663959). @marcoandredinis wrote this part, so perhaps you two could help me rephrasing this comment to be easier to understand?

Reply:
I would phrase it like this

> The database service labels controls the which Database Services (Teleport Agents) are visible to the user. Access to Databases is controlled by the Databased Labels field.

We can add more details if this is not clear.

> The database service labels controls the which Database Services (Teleport Agents) are visible to the user, required when adding Databases in the Enroll New Resource wizard. Access to Databases is controlled by the Databased Labels field.

Reply:
i think marco's first quote box is good enough, we can perhaps add a link to [Database Service](https://goteleport.com/docs/reference/agent-services/database-access-reference/configuration/) that explains what it is too